lib/cli-engine.js

```diff
@@ -338,6 +338,74 @@ function processPath(extensions) {
     };
 }
 
+/**
+ * create a md5Hash of a given string
+ * @param  {string} str the string to calculate the hash for
+ * @returns {string}    the calculated hash
+ */
+function md5Hash(str) {
+    return crypto
+        .createHash("md5")
+        .update(str)
```

Comment:
You need to use the encoding option otherwise it will use `binary` encoding, which you definitely don't want.

```js
.update(str, 'utf8')
```

Or just use my tiny module which handles this: https://github.com/sindresorhus/md5-hex